ros/processor/insights_engine_result_consumer.py

```diff
@@ -1,19 +1,22 @@
+import datetime
 import json
 from ros.lib.app import app, db
 from ros.lib.config import (INSIGHTS_KAFKA_ADDRESS, GROUP_ID,
                             ENGINE_RESULT_TOPIC, get_logger)
-from ros.lib.models import RhAccount, System
-from ros.lib.utils import get_or_create
+from ros.lib.models import RhAccount, System, PerformanceProfile
+from ros.lib.utils import get_or_create, convert_iops_from_percentage
 from confluent_kafka import Consumer, KafkaException
 from ros.processor.metrics import (processor_requests_success,
                                    processor_requests_failures,
                                    kafka_failures)
+from ros.processor.process_archive import get_performance_profile
+from ros.lib.utils import validate_type
 
 SYSTEM_STATES = {
     "INSTANCE_OVERSIZED": "Oversized",
     "INSTANCE_UNDERSIZED": "Undersized",
     "INSTANCE_IDLE": "Idling",
-    "INSTANCE_UNDER_PRESSURE": "Under pressure",
+    "INSTANCE_OPTIMIZED_UNDER_PRESSURE": "Under pressure",
```

Comment:
@apuntamb Noticed the rule hit had this key, INSTANCE_OPTIMIZED_UNDER_PRESSURE, hence the change